net-im/qtox/qtox-9999.ebuild

```diff
@@ -28,6 +28,7 @@ RDEPEND="
 	dev-qt/qtwidgets:5
 	dev-qt/qtxml:5
 	media-gfx/qrencode:=
+	media-libs/libexif
```

Comment:
`:=` since it is using subslots

Reply:
https://github.com/gentoo/gentoo/pull/9869 ;]